src/app/components/apps-view/apps-view.component.ts

```diff
@@ -79,9 +79,7 @@ export class AppsViewComponent implements OnInit {
       { colId: 'resource', colName: 'Resource', colFormatter: CommonUtil.resourceColumnFormatter },
       { colId: 'queueName', colName: 'Queue Name' },
       { colId: 'priority', colName: 'Priority' },
-      { colId: 'partition', colName: 'Partition' },
       { colId: 'nodeId', colName: 'Node ID' },
```

Comment:
Can we move node ID further to the start of the table: key, node, resource and priority.

Reply:
@wilfred-s @yangwwei 

![image](https://user-images.githubusercontent.com/48027290/113126500-4bb01c00-924a-11eb-8f43-1a55541e6f88.png)